### PR TITLE
task/COOKS-243: Update backend/server with recent changes to protx repo

### PR DESCRIPTION
**Pull request by taoteg**
## Overview: ##

This contains all the db queries, view logic and plot rendering code from the api side on the forked codebase.

This is missing an update to downloading resources.  To be done in https://jira.tacc.utexas.edu/browse/COOKS-275

## Related Jira tickets: ##

* [COOKS-243](https://jira.tacc.utexas.edu/browse/COOKS-243)

### CODE REVIEW

File: protx/api.py [hunk 1]
```diff
@@ -83,63 +99,176 @@ def get(self):
...
 
     with engine.connect() as connection:
         result = connection.execute(MALTREATMENT_QUERY)
         data = create_dict(result, level_keys=MALTREATMENT_JSON_STRUCTURE_KEYS)
-
         result = connection.execute(MALTREATMENT_MIN_MAX_QUERY)
         meta = create_dict(result, level_keys=MALTREATMENT_JSON_STRUCTURE_KEYS[:-1])
         return {"data": data, "meta": meta}
+        # Do we need to wrap the response in Flask?
+        # return JsonResponse({"data": data, "meta": meta})
```
Comment (nathanfranklin) [inline protx/api.py:202]: drop

File: protx/api.py [hunk 2]
```diff
@@ -2,9 +2,19 @@
 from sqlalchemy import create_engine
 from protx.log import logging
 from protx.decorators import onboarded_user_required
-from protx.utils.db import (resources_db, create_dict, SQLALCHEMY_DATABASE_URL, SQLALCHEMY_RESOURCES_DATABASE_URL,
-                            DEMOGRAPHICS_JSON_STRUCTURE_KEYS, DEMOGRAPHICS_QUERY, DEMOGRAPHICS_MIN_MAX_QUERY,
-                            MALTREATMENT_JSON_STRUCTURE_KEYS, MALTREATMENT_QUERY, MALTREATMENT_MIN_MAX_QUERY)
+from protx.utils.db import (
+    cooks_db,
```
Comment (nathanfranklin) [inline protx/api.py:6]: drop

File: protx/utils/demographics.py [hunk 3]
```diff
@@ -88,29 +87,28 @@ def get_bin_edges(query_return_df, label_template):
...
-        'SNGPNT': {'col': 6, 'range': (0, 100), 'label_fmt': not_currency, 'color': '#FFA15A'}
-    },
-    'hu': {
-        'CROWD': {'col': 8, 'range': (0, 100), 'label_fmt': not_currency, 'color': '#19D3F3'}
-    },
-    'dollars': {
-        'PCI': {'col': 10, 'range': None, 'label_fmt': currency, 'color': '#FF6692'}
-    }
-}
+# subplot_mapping_aes = {
```
Comment (nathanfranklin) [inline protx/utils/demographics.py:95]: can we just drop these lines?

**Review by nathanfranklin — APPROVED** [review]
LGTM 👍 

Download-fixes can come in separate PR

File: protx/utils/demographics.py (hunk 3 above)
Comment (taoteg) [reply protx/utils/demographics.py]: Done!

**Pull request merged**